composition/book.rb

```diff
@@ -0,0 +1,12 @@
+require 'securerandom'
+
+class Book
+  attr_reader :id, :rentals
```

Comment:
- [ ] Based on the requirement, kindly create a setter method for the rentals attribute. You can use this shorthand method 🖱️ 💻:

```ruby
attr_accessor :rentals
```